examples/v2stream/v2stream.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
hello, i am testing this example with your branch of the `alpaca-trade-api-go` package  and receive this error:
```
$ go run main.go 
2021/02/28 20:01:41 failed to open Alpaca data stream: failed to WebSocket dial: expected handshake response status code 101 but got 403
2021/02/28 20:01:42 failed to open Alpaca data stream: failed to WebSocket dial: expected handshake response status code 101 but got 403
2021/02/28 20:01:43 failed to open Alpaca data stream: failed to WebSocket dial: expected handshake response status code 101 but got 403
panic: failed to WebSocket dial: expected handshake response status code 101 but got 403

goroutine 1 [running]:
main.main()
        /main.go:32 +0x245
exit status 2
```

Can you double check this is working as expected?

Reply:
Hey ignore me, i checked out the wrong commit from your branch.  Testing looks good 👍

Reply:
Can you please do the review, too?